### DISCUSSION TITLE
Play Music from more than just first station of ride

### DISCUSSION THREAD

**Discussion by 733737** [OpenRCT2/OpenRCT2#18048] [Feature requests]
currently when you select music to be played from a ride, it only plays music on station 1, station 2 and up do not play music.

This really bothers me, as I built a ride with a 3rd station near the back of my park and the 1st station near the front of the park, so the station near the back of the park has no music

**Comment by TheClaudeQC** [comment]
Yeah, it could be cool to have the music per stations, so we can implement same song or different songs. If you want the same in the entire park, you can check a "synchronise whit 1st station" box.

**Comment by 733737** [comment]
#9605
